manifests/init.pp

```diff
@@ -67,28 +75,30 @@
     ensure => 'installed',
   }
 
-  if $manage_init_script == true {
+  # Don't manage init script for Ubuntu
+  if ($manage_init_script == true) and ($::osfamily != 'Ubuntu') {
     file { '/etc/init.d/tac_plus':
       ensure  => 'file',
       content => template($init_template),
       owner   => 'root',
       group   => 'root',
       mode    => '0744',
-      before  => Service['tac_plus'],
+      before  => Service[$tac_plus_service],
     }
   }
 
   # TODO: what about the mode?
-  file { '/etc/tac_plus.conf':
+  file { $tac_plus_config:
```

Comment:
Please use a generic name like ```tacas_plus_config``` and use the ```path``` attribute of the file resource to specify the actual file path. With this you can reference the generic resource name in later dependencies. Makes it easier to read and code.